### ISSUE TITLE
Token popup truncate the content

### ISSUE THREAD

**Issue by samuelsov** [veda-consulting/uk.co.vedaconsulting.mosaico#136]
Minor aesthetic bug but when using the Tokens link, a popup that opens seems to have a fixed width which doesn't match the content as seen on this screenshot :

![token-popup-display](https://cloud.githubusercontent.com/assets/372004/24914021/f7794c5e-1ea0-11e7-9fc9-2f9558b5aa7c.png)

**Comment by mlutfy** [comment]
I worked around this by changing:

./js/tinymce-plugins/civicrmtoken/plugin.js

```
    win = editor.windowManager.open({
      title: 'Insert Token',
      autoScroll: true, // Can we move this to the grid?
      width: 900, // <-----
      height: 600, // <-----
      body: [
```

I found 900px by 600px to be a good dimension. On most sites, the tokens can be pretty long.

**Comment by totten** [comment]
@samuelsov @mlutfy I've merged #213 which makes the size more adaptive (and, in the typical case, has the effect of making the window wider).
